bin/fapolicy-analyzer

```diff
@@ -2,22 +2,35 @@
 
 # Initialize globals
 pidAgent=0
+strCallbackFile=/tmp/fileCallback.txt
```

Comment:
Let's use a more descriptive name on the temp file here

Reply:
Modified to "/tmp/fapolicy-analyzer-dbus-callback.tmp"